Pages/itemPage.js

```diff
@@ -0,0 +1,55 @@
+class ItemPage {
+    constructor (){
+        this.title = "ItemPage";
+    }
+    get bidMessage (){
```

Comment:
Before getters add comment "GETTERS", and before actions add comment "ACTIONS" to make code more readable. Apply this for all page objects.
Also, before every method inside your page object add console log with description what is happening in that method. Apply this for all page objects.

Reply:
I added comments for getters and setters. I couldn't call console.log() from inside of the class, so I added comment before every method with description

Reply:
I removed comments, and added console.log()